feature/experimental/qos/ate_tests/qos_output_queue_counters_test/qos_output_queue_counters_test.go

```diff
@@ -230,6 +230,51 @@ func ConfigureQoS(t *testing.T, dut *ondatra.DUTDevice) {
 	d := &oc.Root{}
 	q := d.GetOrCreateQos()
 
+	t.Logf("Create qos forwarding groups config")
+	forwardingGroups := []struct {
```

Comment:
desc is unused and targetGroup (which is misspelled here) can just be derived from the queue name. Might as well simplify this to?

queues := []string {"BE0", "BE1", "AF1", "AF2", "AF3", "AF4", "NC1"}
t.Logf("qos forwarding groups queues: %v", queues)
for _, queue := range queues {
  targetGroup := fmt.Sprintf("target-group-%s", queue)
 ...
}

Reply:
When are not addressed, please say why they are not being addressed before resolving.

Reply:
Didn't address for a couple of reasons:
- Similar blocks of code are used in other places in the script. For e.g. around line # 524, for 'schedulerInts'. Better to maintain consistency across the script.
- Not directly related to the main purpose of this Pull Request (which was to fix the order of pushing different configuration blocks to resolve dependency issues)

Let me know. Thanks!